src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -117,32 +119,30 @@ class TizenControls extends Controls {
             controlbar.nextUpToolTip = nextUpToolTip;
 
             // NextUp needs to be behind the controlbar to not block other tooltips
-            this.div.appendChild(nextUpToolTip.element());
+            element.appendChild(nextUpToolTip.element());
         }
 
-        this.div.appendChild(controlbar.element());
+        element.appendChild(controlbar.element());
 
         // Seekbar
-        const seekbar = this.seekbar = new TizenSeekbar(model, api, this.controlbar.elements.time);
-        this.div.appendChild(seekbar.element());
+        const seekbar = this.seekbar = new TizenSeekbar(model, api, controlbar.elements.time);
+        element.appendChild(seekbar.element());
 
         // Settings/Tracks Menu
         const localization = model.get('localization');
         const settingsMenu = this.settingsMenu = new TizenMenu(api, model.player, this.controlbar, localization);
```

Comment:
`this.settingsMenu` will be overridden in `super.enable`. Deal with it like you do `controlbar` (making sure to destroy the one created in the base controller first).